Registry/Registry.EM_COMMON

```diff
@@ -3006,9 +3046,13 @@ package   swintopt2      swint_opt==2               -              state:swdown2
 package   aercuopt       aercu_used==1              -              state:aeromcu,CU_UAF,EFCS,EFIS,EFSS,qr_cu,qs_cu,nc_cu,ni_cu,nr_cu,ns_cu,ccn_cu,aerovar;aerocu:cu_sulfate,cu_seasalt,cu_dust1,cu_dust2,cu_dust3,cu_dust4,cu_phoocar,cu_phiocar,cu_phobcar,cu_phibcar
 
 package   slopeopt       slope_rad==1               -              -
-package   gwd_used       gwd_used==1                -              state:oc12d,oa1,oa2,oa3,oa4,ol1,ol2,ol3,ol4,dtaux3d,dtauy3d,dusfcg,dvsfcg
+package   gwd_used_1     gwd_used_1==1              -              state:oc12d,oa1,oa2,oa3,oa4,ol1,ol2,ol3,ol4,dtaux3d,dtauy3d,dusfcg,dvsfcg
+package   gwd_used_3     gwd_used_3==1              -              state:var2dls,oc12dls,oa1ls,oa2ls,oa3ls,oa4ls,ol1ls,ol2ls,ol3ls,ol4ls,var2dss,oc12dss,oa1ss,oa2ss,oa3ss,oa4ss,ol1ss,ol2ss,ol3ss,ol4ss 
+package   gwd_used_33    gwd_used_33==1             -              state:var2dls,oc12dls,oa1ls,oa2ls,oa3ls,oa4ls,ol1ls,ol2ls,ol3ls,ol4ls,var2dss,oc12dss,oa1ss,oa2ss,oa3ss,oa4ss,ol1ss,ol2ss,ol3ss,ol4ss,dtaux3d_ls,dtauy3d_ls,dtaux3d_bl,dtauy3d_bl,dtaux3d_ss,dtauy3d_ss,dtaux3d_fd,dtauy3d_fd,dusfcg_ls,dvsfcg_ls,dusfcg_bl,dvsfcg_bl,dusfcg_ss,dvsfcg_ss,dusfcg_fd,dvsfcg_fd
 package   nogwdopt       gwd_opt==0                 -              -
 package   gwdopt         gwd_opt==1                 -              -
+package   gwdopt_gsl     gwd_opt==3                 -              -
+package   gwdopt_gsl_dev gwd_opt==33                -              -
```

Comment:
Maybe these two lines should be removed if gwd_used_1/3/33 are there?

Reply:
I'll admit I'm not 100% clear on what lines 3052-3055 do.  Lines 3052-3053 already existed for gwd_opt={0,1} so I just extended the pattern to lines 3054-3055 by including gwd_opt={3,33}.  If we don't need lines 3054-3055, then do we need 3052-3053?

Reply:
This is where we are supposed to package up the zillion new fields that are used just for this scheme. Look a few lines up and down. The second dash is to be replaced by "state:dtaux3d_ls,..."

This is a commas separated list of the new fields.

Reply:
I remember that's how it was done for the gwd options prior to v4.2.  Do we still need these four lines, or can they be deleted?  (I haven't tried deleting them and seeing if it compiles.)